rhizome/host/bin/prep_host.rb

```diff
@@ -106,3 +113,27 @@
 
 r "systemctl enable cron"
 r "systemctl start cron"
+
+# setup blocking unused ip addresses
+FileUtils.touch(Util::NFTABLES_PATH)
+block_ip_rules = []
+if (to_block_ip_ranges = JSON.parse(additional_ip_addresses))
+  to_block_ip_ranges.each do |range|
+    net = NetAddr::IPv4Net.parse(range)
+    0.upto(net.len - 1) do |i|
+      block_ip_rules.push("ip daddr #{net.nth(i)} drop")
+    end
+  end
+end
+safe_write_to_file(Util::NFTABLES_PATH, <<SETUP_ADDITIONAL_IP_BLOCKING)
+table inet drop_unused_ip_packets;
+delete table inet drop_unused_ip_packets;
+table inet drop_unused_ip_packets {
+  chain prerouting {
+    type filter hook prerouting priority 0; policy accept;
+    #{block_ip_rules.join("\n")}
```

Comment:
Block is a pretty loaded word to use here. There is a block of cheese but also to block a door. I take it you mean the former.

Reply:
I have updated it, I actually meant the latter one.